user-guide/Advanced_Modules/User-Defined_APIs/UD_APIs_Triggering_an_API.md

```diff
@@ -144,7 +145,7 @@ The *errorCode* field of an error contains an error code that can be used by the
 | MessageBrokerNotInitialized | 1006 | 503 | The message broker and session used to connect to NATS have not been initialized yet. |
 | FailedToReadBody | 1007 | 500 | An exception was thrown while reading the request body. |
 | AuthenticationHeaderInvalid | 1008 | 401 | The HTTP [authentication](#authentication) header is not valid. Make sure the token is passed as a `Bearer` token. |
-| BodyTooLarge | 1009 | 413 | The body size is limited to 30 MB. This error will be thrown if the size is larger than that. |
+| BodyTooLarge | 1009 | 413 | The body size is limited to 29 MB. This error will be thrown if the size is larger than that. |
```

Comment:
S: This error will be thrown if the reply of the script has a size larger than that.

Reply:
This is the error for the input body, it incorrectly referred to '30MB'. The new one for the response is at line 139.